packages/discordx/src/decorators/decorators/Slash.ts

```diff
@@ -40,6 +44,8 @@ export function Slash<T extends string>(
  */
 export function Slash(options?: ApplicationCommandOptions): MethodDecoratorEx {
   return function <T>(target: Record<string, T>, key: string) {
+    SlashNameValidator(options?.name ?? key);
```

Comment:
Just a small clean code thing:
```suggestion
    const name = options?.name ?? key;
    SlashNameValidator(name);
```

and of course, replace name below with the new var